bfxapi/rest/bfx_rest.py

```diff
@@ -163,20 +163,21 @@ async def get_public_ticker(self, symbol):
         ticker = await self.fetch(endpoint)
         return ticker
 
-    async def get_public_tickers(self, symbols):
+    async def get_public_tickers_history(self, symbols):
```

Comment:
Both functions exists one under the other

Reply:
Please check the source code

Reply:
ok, so we have: 

 1) LOC 152 - https://docs.bitfinex.com/reference#rest-public-ticker `https://api-pub.bitfinex.com/v2/ticker/Symbol`
 2) ticker*s* history, added here in replacement for (3)- https://docs.bitfinex.com/reference#tickers-history - `https://api-pub.bitfinex.com/v2/tickers/hist`
3) `get_public_tickers`, deleted, was https://docs.bitfinex.com/reference#rest-public-tickers - `https://api-pub.bitfinex.com/v2/tickers` (note the s in tickers)

i think we should readd the now missing `get_public_tickers`

Reply:
You're right 🤦